src/status_im/ui/screens/wallet_connect/session_proposal/views.cljs

```diff
@@ -66,17 +66,23 @@
                          :background-color (:color @selected-account)
                          :on-press on-press}]]))
 
-(defview success-sheet-view [{:keys [topic]}]
+(defview success-sheet-view [{:keys [wc-version] :as session-data}]
   (letsubs [visible-accounts @(re-frame/subscribe [:visible-accounts-without-watch-only])
             dapps-account [:dapps-account]
             sessions [:wallet-connect/sessions]
+            sessions-legacy [:wallet-connect-legacy/sessions]
             managed-session [:wallet-connect/session-managed]]
-    (let [{:keys [peer state] :as session} (first (filter #(= (:topic %) topic) sessions))
-          {:keys [accounts]} state
+    (let [{:keys [topic]} (when-not (= wc-version 1) session-data)
```

Comment:
Would be nice to have that `1` in some constant, but also not a big deal I suppose. It just might be simpler when it will come to removing all those conditions in case if those are spread in file or different namespaces.